dev/com.ibm.ws.microprofile.metrics.common/resources/com/ibm/ws/microprofile/metrics/resources/Metrics.nlsprops

```diff
@@ -59,6 +59,11 @@ internal.error.CWMMC0006E=CWMMC0006E: An internal error occurred.
 internal.error.CWMMC0006E.explanation=The application server experienced an internal error.
 internal.error.CWMMC0006E.useraction=Gather a set of traces and open a new issue against your application server.
 
+#User configured mpMetrics to use user-defined library for Micrometer binaries without Prometheus Registry JAR
+noPrometheusRegistry.info.CWMMC0008I=CWMMC0008I: The /metrics endpoint is not available because no Prometheus Registry is available.
+noPrometheusRegistry.info.CWMMC0008I.explanation=The mp.metrics.prometheus.enabled MicroProfile Config property might be set to false. Alternatively, the MicroProfile Metrics feature might be loading Micrometer libraries that are defined through the libraryRef attribute, but a referenced Library does not contain a Micrometer Prometheus Registry.
```

Comment:
The message and the explanation don't seem to match.  Message suggests the reason the metrics endpoint is not available is due to no registry being available, but explanation suggests it could also be due to configuration.  If the message is correct, perhaps we need a second message for the case where the /metrics endpoint isn't available due to configuration.